redis/Cargo.toml

```diff
@@ -100,7 +100,7 @@ cluster = ["crc16", "rand"]
 script = ["sha1_smol"]
 tls-native-tls = ["native-tls"]
 tls-rustls = ["rustls", "rustls-native-certs", "rustls-pemfile", "rustls-pki-types"]
-tls-rustls-insecure = ["tls-rustls"]
+tls-rustls-insecure = ["tls-rustls", "rustls/ring"]
```

Comment:
This doesn't make sense, why did you pull in `rustls/ring` only for the `-insecure` variant?

Reply:
because of this line that you've added - https://github.com/redis-rs/redis-rs/blob/3cf579fd0da327d018559f1981e6cc27d6fe9cb3/redis/src/connection.rs#L906

Reply:
can it be replaced with something generic?